index.html

```diff
@@ -117,12 +117,16 @@ <h5 class="subheader">Bio-Formats is a Java library for reading and writing imag
         <hr class="whitespace">
         <div class="row column text-center">
             <h2><img src="{{ site.baseurl }}/img/logos/ome-files.svg" alt="OME Files logo" class="section-title-logo" /></h2>
-            <h5 class="subheader">OME Files is a reference implementation of the OME data model and the OME-TIFF file format for developers to use in their own software.</h5>
+            <h5 class="subheader">OME Files is a reference implementation of the OME data model and the OME-TIFF file format. OME also organises the OME-NGFF project, a community effort to develop "Next Generation File Formats".</h5>
         </div>
         <hr>
         <div class="row column text-center">
-            <p>OME Files is a package of tools for reading and writing the standard OME-TIFF file format and working with the OME data model, which builds upon what began as a native C++ implementation of Bio-Formats. Potential uses include export of images using OME-TIFF, saving of acquired image data in OME-TIFF, reading metadata and image data from OME-TIFF for visualization and analysis, or use of the data model metadata APIs for handling metadata.</p>
-            <a href="{{ site.baseurl }}/ome-files/">Get started with OME Files <i class="fa fa-long-arrow-right"></i></a>
+            <p>
+              OME Files is a package of tools for reading and writing the standard OME-TIFF file format and working with the OME data model, which builds upon what began as a native C++ implementation of Bio-Formats.
```

Comment:
Why to cite the C++ beginnings ? Is that important ? Seems just confusing bit of irrelevant history ?

Reply:
resolved